addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/handler/HeliosHandler213.java

```diff
@@ -0,0 +1,962 @@
+            }
+
+            if (response != null) {
+                JsonObject jsonObject = parser.parse(response.readEntity(String.class)).getAsJsonObject();
+                if (jsonObject.get("success").toString().equals("true")) {
+                    logger.trace("Successfully pulled log entries from the Helios IP Vario '{}'",
+                            getThing().getUID().toString());
+                    JsonObject js = (JsonObject) jsonObject.get("result");
+                    RESTEvent[] eventArray = gson.fromJson(js.getAsJsonArray("events"), RESTEvent[].class);
+                    return Arrays.asList(eventArray);
+                } else {
+                    RESTError error = gson.fromJson(jsonObject.get("error").toString(), RESTError.class);
+                    logger.error(
+                            "An error occurred while communicating with the Helios IP Vario '{}' : code '{}', param '{}' : '{}'",
+                            new Object[] { getThing().getUID().toString(), error.code, error.param,
+                                    error.description });
+                    this.logSubscriptionID = 0;
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+                    scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+                    return null;
+                }
+            } else {
+                logger.warn("An error occurred while polling log entries from the Helios IP Vario '{}'",
+                        getThing().getUID().toString());
+                this.logSubscriptionID = 0;
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR);
+                scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+                return null;
+            }
+        }
+
+        return null;
+    }
+
+    private List<RESTSwitch> getSwitches() {
+        switchTarget = baseTarget.path(SWITCH_PATH);
+
+        Response response = null;
+        try {
+            response = switchTarget.resolveTemplate("ip", ipAddress).resolveTemplate("cmd", CAPABILITIES)
+                    .request(MediaType.APPLICATION_JSON_TYPE).get();
+        } catch (Exception e) {
+            logger.error(
+                    "An exception occurred while requesting switch capabilities from the Helios IP Vario '{}' : '{}'",
+                    getThing().getUID().toString(), e.getMessage());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+            scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+            return null;
+        }
+
+        if (response != null) {
+            JsonObject jsonObject = parser.parse(response.readEntity(String.class)).getAsJsonObject();
+            if (jsonObject.get("success").toString().equals("true")) {
+                logger.info("Successfully requested switch capabilities from the Helios IP Vario '{}'",
```

Comment:
Please change the logging level to `debug`. `trace` can be used for even more details, where necessary.